src/main/java/com/dezorganizacja/bridge/view/graphical/StageProperties.java

```diff
@@ -0,0 +1,30 @@
+package com.dezorganizacja.bridge.view.graphical;
+
+import javafx.stage.Stage;
+
+public class StageProperties {
```

Comment:
On Tue, Apr 28, 2015 at 12:39:21PM -0700, Marek Rusinowski wrote:
> > @@ -0,0 +1,30 @@
> > +package com.dezorganizacja.bridge.view.graphical;
> > +
> > +import javafx.stage.Stage;
> > +
> > +public class StageProperties {
> 
> Nie rozumiem jaki jest cel tej klasy, przeczytałem kod, ale nie mogę zrozumieć zamysłu.

Stage podczas zmiany scen nie trzyma właściwości, jak rozmiar okna,
maksymalizacja itp. Ta klasa właśnie zapamiętuje stan stage-a i
przenosi te ustawienia na następną scenę.

Uznałem, że ładniej będzie stworzyć klasę, która tym będzie się
zajmowała, niż robić to na chama w .loadMainScene(). Według mnie
dowalenie tego wszystkiego tam, zaśmieciłoby tylko kod.

-- 
Bartłomiej Puget